### ISSUE TITLE
Skip lesions on temporal artery biopsy

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9923]
**Preferred term label:**
Skip lesions on temporal artery biopsy

**Synonyms**
Segmental inflammation

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Giant cell arteritis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as
HP:0430105

Skip lesions on temporal artery biopsy
